jenkins/DockerfileBlueocean

```diff
@@ -0,0 +1,42 @@
+FROM jenkinsci/blueocean:1.3.5
+USER root
+ENV PACKAGES "unzip curl openssl ca-certificates git ruby ruby-json libc6-compat"
+ENV CF_CLI_VERSION "6.26.0"
+ENV CF_BGD_VERSION "1.1.0"
+ENV CF_BGD_CHECKSUM "fe6ebd3c2dc3a287db0d31eeaed200d1a27117d9a6ddd875de561e4a6e8858d1"
+ENV CF_BGD_TEMPFILE "/tmp/blue-green-deploy.linux64"
+ENV CF_AUTOPILOT_VERSION "0.0.3"
+ENV CF_AUTOPILOT_CHECKSUM "c3b5a38ba7a9817e12d6ab4c98418d707c49c1d3eb68e5a79e701531bd8fa1cd"
+ENV CF_AUTOPILOT_TEMPFILE "/tmp/autopilot-linux"
+
+RUN apk add --update $PACKAGES && rm -rf /var/cache/apk/*
+
+RUN curl -L "https://cli.run.pivotal.io/stable?release=linux64-binary&version=${CF_CLI_VERSION}" | tar -zx -C /usr/local/bin
+RUN curl -L 'https://github.com/stedolan/jq/releases/download/jq-1.5/jq-linux64' -o /usr/local/bin/jq && chmod +x /usr/local/bin/jq
+
+RUN curl -L -o "${CF_BGD_TEMPFILE}" \
+  "https://github.com/bluemixgaragelondon/cf-blue-green-deploy/releases/download/v${CF_BGD_VERSION}/blue-green-deploy.linux64" \
+  && echo "${CF_BGD_CHECKSUM}  ${CF_BGD_TEMPFILE}" | sha256sum -c - \
+  && chmod +x "${CF_BGD_TEMPFILE}" \
+  && cf install-plugin -f "${CF_BGD_TEMPFILE}" \
+  && rm "${CF_BGD_TEMPFILE}"
+
+RUN curl -L -o "${CF_AUTOPILOT_TEMPFILE}" \
+  "https://github.com/contraband/autopilot/releases/download/${CF_AUTOPILOT_VERSION}/autopilot-linux" \
+  && echo "${CF_AUTOPILOT_CHECKSUM}  ${CF_AUTOPILOT_TEMPFILE}" | sha256sum -c - \
+  && chmod +x "${CF_AUTOPILOT_TEMPFILE}" \
+  && cf install-plugin -f "${CF_AUTOPILOT_TEMPFILE}" \
+  && rm "${CF_AUTOPILOT_TEMPFILE}"
+
+# now install docker
+
+RUN apk add --update curl bash \
+  && rm -rf /var/cache/apk/* \
+  && curl -L https://get.docker.com/builds/Linux/x86_64/docker-latest > /usr/bin/docker \
+  && chmod +x /usr/bin/docker
+ADD https://download.docker.com/linux/static/stable/x86_64/docker-17.12.0-ce.tgz .
```

Comment:
Are we not adding docker twice here with `curl -L https://get.docker.com/builds/Linux/x86_64/docker-latest > /usr/bin/docker` and `ADD https://download.docker.com/linux/static/stable/x86_64/docker-17.12.0-ce.tgz .`?

Reply:
Thanks Ben, yep you're right. I'll fix that and add docker-compose too.